src/stk/calculators/energy/energy_calculators.py

```diff
@@ -128,29 +130,12 @@ def inner(self, mol):
     return inner
 
 
-class EnergyCalculator:
+class EnergyCalculator(Calculator):
     """
     Calculates the energy of molecules.
 
     """
 
-    def __init__(self, use_cache=False):
```

Comment:
This may be a stupid question, but what happens to all the classes that inherit from EnergyCalculator but call 
```python 
super().__init__(some_args)
```
If the __init__ method is removed, do they break?

Reply:
Or is the __init__ inherited from the `Calculator` class?

Reply:
yep, you keep searching up until you hit something, in this case they will hit the init of the Calculator class.